litmus-portal/frontend/src/views/AnalyticsDashboard/KubernetesDashboards/Table/TableData.tsx

```diff
@@ -1,42 +1,18 @@
 /* eslint-disable no-unused-expressions */
 /* eslint-disable prefer-destructuring */
 /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
please remove the above and refrain from using these linting disable commands further. If you have these in other places, I would request you to remove them too. If in any case, it is unavoidable...you can consult me first.

Reply:
Done